simulator/docs/env-variables.md

```diff
@@ -6,6 +6,9 @@ used to configure kube-scheduler-simulator.
 Please refer [docker-compose.yml](./../../docker-compose.yml) as an example
 use.
 
+## For Simulator
+This part is deprecated. Please refer to the simulator [config.yml](./../config.yml).
```

Comment:
Add the note like "We're planning to remove the configuration via environment variables."

Reply:
Also, could you add some note on README.md or somewhere new in /docs about config.yml?